integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/BridgeSteps.java

```diff
@@ -29,73 +31,83 @@ public BridgeSteps(TestContext context) {
         this.context = context;
     }
 
-    @Given("get list of Bridge instances fails with HTTP response code (\\d+)$")
-    public void authenticationIsEnabled(int responseCode) {
+    @Given("^the list of Bridge instances fails with HTTP response code (\\d+)$")
+    public void listOfBridgeInstancesReturnsHTTPResponseCode(int responseCode) {
         BridgeResource.getBridgeListResponse(context.getManagerToken())
                 .then()
                 .statusCode(responseCode);
     }
 
-    @When("create a Bridge with randomly generated name with access token")
-    public void createRandomBridge() {
-        String bridgeName = Utils.generateId("bridge");
+    @When("^a new Bridge \"([^\"]*)\" is created$")
```

Comment:
I prefer active sentence for actions invoked by test steps and passive sentences to indicate a checking step.
What about something like `"^create a new Bridge \"([^\"]*)\"$"`

Reply:
sure.
gonna review the steps

Reply:
done